### ISSUE TITLE
Comments Missing

### ISSUE THREAD

**Issue by Cekis** [Rohansi/SWG-ScriptConverter#1]
Would love it if you could take the original files, grab the comments, and insert them into their respective position in the new Java files.  Obviously, we've progressed beyond a point that they're really needed, but they are useful from time to time.

By the way, thank you for creating this originally!  For those projects that did migrate to Java from their .script equivalent versions it created a very solid .script to Java conversion.

**Comment by Rohansi** [comment]
Sorry for the late reply! By new Java files do you mean the output from this program or the ones being used by the private server? It'd be relatively easy to add to the output here but I think adding them to the private server's copy might be better if done as a git merge after reconverting the scripts.

**Comment by Cekis** [comment]
Ah, that's actually a great idea.  What I originally meant was adding them to the "server's copy".  Which, by the way, isn't an active server.  We took your original output from 2014-ish and intended to open a server with it, but didn't.  Instead, I pushed to "open" the work we did and the result has been "SWG-Source".  We shared all of our bug fixes, additions, etc. with anybody that wants to add them to their server (private or public).  It's now a progressing repo:  [https://github.com/SWG-Source/](url).

**Comment by Rohansi** [comment]
Very cool! I've never actually got to try the game before so I might have to give it a go later.

**Comment by Cekis** [comment]
I figured you probably hadn't tried it, but what I wanted to say was that without your conversion, it would have been much more difficult for people to work on it.  Was thinking you may want to see how far it has come via commit history and such and just how long your "output" has been in use.  But playing the game is a good option too, lol!  At any rate, I plan on using your tool to do what you suggested.  Hopefully things have not changed that much and it can still happen.

**Comment by Rohansi** [comment]
Yeah, it's nice seeing what came of this. I didn't really know the person that asked me if I could complete this and never kept in touch with him after so it was silence until I dug around and found that repo. Keep up the good work!

**Comment by Rohansi** [comment]
@Cekis I meant to ask, did anyone end up making fixes to this code? I might start on this today or next weekend, but it'd be good to bring changes back into this repo first!

**Comment by Cekis** [comment]
Sorry was on vacation.  I don't believe anybody made changes?  It's possible as we didn't do the full switch from script to java until later (2016 ish), but I doubt there were any changes.

**Comment by Rohansi** [comment]
I guess we'll find out when it's time to merge then!

**Comment by Cekis** [comment]
Excited to see the output!

**Comment by rohvani** [comment]
Just some background: the people behind this request was the original SWG Reborn team (myself/seefo, aconite, light, etc).  This tool was developed prior to @apathyboy's python-based transpiler being released alongside the rest of swgreveniens work (in the swgmasters repos).

@Rohansi Just wanted to clarify, since we never moved forward with your tool, due to issues with many of the scripts not being properly converted (even though they all compiled), so these issues would need to be fixed before @Cekis could use this.

@apathyboy's transpiler may be found here: https://bitbucket.org/swgmasters/swg-src/src/master/utils/mocha/
